apps/to-do-api/eslint.config.mjs

```diff
@@ -0,0 +1,15 @@
+import lintRules from "@pagopa/eslint-config";
+
+export default [
+  ...lintRules,
+  {
+    rules: {
+      // The following rules are raising errors when running the lint command
+      "@typescript-eslint/no-empty-function": "off",
+      "@typescript-eslint/no-unused-expressions": "off",
+    },
```

Comment:
**note**: without this block, when running the `lint` script, eslint is throwing an error.

Errors:
```
TypeError: Error while loading rule '@typescript-eslint/no-empty-function': Cannot read properties of undefined (reading 'allow')

TypeError: Error while loading rule '@typescript-eslint/no-unused-expressions': Cannot read properties of undefined (reading 'allowShortCircuit')
```

Reply:
can you try with `eslint 8`?

Reply:
it looks like `eslint` on version 8 is working. Check PR #31